### PR TITLE
Update the landing page for Porter Docs removing the porter CLI commands

### PR DESCRIPTION
**Pull request by iennae**
# What does this change
Cleans up the Porter Docs landing page. 

# What issue does it fix
Documentation update. 

# Checklist

- [x] Documentation

### CODE REVIEW

File: docs/content/docs.md [hunk 1]
```diff
@@ -3,59 +3,14 @@ title: Porter Docs
...
 
-$ porter build
-created Dockerfile
-created bundle.json
-created cnab/app/run
+Porter has a lot of documentation and we're in the process of reorganizing and updating it. We welcome [contributions](/contribute/) from the community. 
 
-$ porter install
-```
+## Quicklinks
```
Comment (carolynvs) [inline docs/content/docs.md:12]: Do you mind if we continue calling this section "Next Steps"? My goal was to have a section like this at the bottom of pages that lead readers to good things to read next. Quicklinks doesn't quite imply the same thing.

Comment (iennae) [reply docs/content/docs.md:12]: What about having both? As the landing page I was thinking this is like Step 0 so here are a set of areas you could go to versus "Next"

Comment (carolynvs) [reply docs/content/docs.md:12]: Ah yeah I see how the main docs page is difference. Makes sense. 👍

**Review by carolynvs — APPROVED** [review]
LGTM

**Pull request merged**